config/controller.yaml

```diff
@@ -31,6 +32,8 @@ spec:
         cluster-autoscaler.kubernetes.io/safe-to-evict: "false"
       labels:
         app: tekton-triggers-controller
+        triggers.tekton.dev/release: "devel"
```

Comment:
Would you mind adding a comment explaining? e.g. https://github.com/tektoncd/pipeline/blob/master/config/controller.yaml#L36

Reply:
Resolved